.docker.env

```diff
@@ -12,4 +12,4 @@ GOOGLE_PROJECT_ID=logflare-dev-238720
 GOOGLE_PROJECT_NUMBER=1023172132421
 LOGFLARE_GRPC_PORT=50051
 
-POSTGRES_BACKEND_URL=postgresql://postgres:postgres@db:5432/logflare_docker
+LOGFLARE_NODE_HOST=127.0.0.1
```

Comment:
We probably should add EditorConfig or something to force trailing newline on last line.